### ISSUE TITLE
Finish Wyckoff functionality in cif parsing

### ISSUE THREAD

**Issue by kyledmiller** [materialsproject/pymatgen#2322]
**Is your feature request related to a problem? Please describe.**
I would like to use Pymatgen's wonderful structure manipulation functionality but cif parsing/manipulating/rewriting pipeline in Pymatgen doesn't preserve the wyckoff letters and multiplicities. I see in the code that it's planned for implementation in `site_properties`, that would be great.

**Describe the solution you'd like**
Parse wyckoff letters and multiplicities from cifs and write them to cifs.

**Comment by shyuep** [comment]
You can use the SpaceGroupAnalyzer.get_symmetrized_structure to get the Wyckoff symbols.

**Comment by mkhorton** [comment]
I think this issue is more about propagating metadata already existing in the CIF, i.e. for Wyckoff symbols that have already been annotated, these could be parsed as site properties.

**Comment by kyledmiller** [comment]
Yes, indeed the get_symmetrized structure works well if I want to find the Wyckoff symbols using spglib but as @mkhorton suggested, I desire to propagate existing metadata in CIFs produced by AFLOW's prototypying code.

**Comment by shyuep** [comment]
As far as I know, that information is not parsed today. But if someone wants to take a crack at it, I am happy to include it.

**Comment by kyledmiller** [comment]
Sounds good. I might try my hand in the coming months.

**Comment by CompRhys** [comment]
@kyledmiller I have some not perfect code that might do what you want re AFLOW prototypes here https://github.com/CompRhys/aviary/blob/main/aviary/wren/utils.py
